components/infobox/commons/infobox_league.lua

```diff
@@ -29,97 +37,102 @@ end
 
 function League:createInfobox()
 	local args = self.args
+	local frame = mw.getCurrentFrame()
+	local links
 
 	-- set Variables here already so they are available in functions
 	-- we call from here on, e.g. createPrizepool
 	self:_definePageVariables(args)
 
-	self.infobox:name(args.name)
-				:image(args.image, args.default)
-				:centeredCell(args.caption)
-				:header('League Information', true)
-				:fcell(Cell:new('Series')
-					:options({})
-					:content(
-						self:_createSeries(mw.getCurrentFrame(), args.series, args.abbreviation),
-						self:_createSeries(mw.getCurrentFrame(), args.series2, args.abbreviation2)
-					)
-					:make()
-				)
-				:fcell(Cell:new('Organizer(s)')
-					:options({})
-					:content(
-						unpack(self:_createOrganizers(args))
-					)
-					:make()
-				)
-				:fcell(Cell:new('Sponsor(s)')
-					:options({})
-					:content(
-						args.sponsor or args.sponsor1,
-						args.sponsor2,
-						args.sponsor3,
-						args.sponsor4,
-						args.sponsor5
-					)
-					:make()
-				)
-				:cell('Server', self:getServer(args))
-				:fcell(Cell:new('Type')
-					:options({})
-					:content(args.type:sub(1,1):upper()..args.type:sub(2))
-					:categories(
-						function(_, ...)
-							local value = select(1, ...)
-							value = tostring(value):lower()
-							if value == 'offline' then
-								self.infobox:categories('Offline Tournaments')
-							elseif value == 'online' then
-								self.infobox:categories('Online Tournaments')
-							else
-								self.infobox:categories('Unknown Type Tournaments')
-							end
-						end
-					)
-					:make()
-				)
-				:cell('Location', self:_createLocation({
+	local widgets = {
+		Header{name = args.name, image = args.image},
+		Center{content = {args.caption}},
+		Title{name = 'League Information'},
+		Cell{
+			name = 'Series',
+			content = {
+				self:_createSeries(frame, args.series, args.abbreviation),
+				self:_createSeries(frame, args.series2, args.abbreviation2)
+			}
+		},
+		Cell{name = 'Organizer(s)', content = self:_createOrganizers(args)},
+		Cell{name = 'Sponsor(s)', content = self:getAllArgsForBase(args, 'sponsor')},
```

Comment:
ageofempires would like to override the content of sponsors, so having that one in a customizable would be great.
Organizer is not actually needed in that Customizable, so could be left outside.
```suggestion
		Customizable{
			id = 'org',
			children = {
				Cell{name = 'Organizer(s)', content = self:_createOrganizers(args)},
				Cell{name = 'Sponsor(s)', content = self:getAllArgsForBase(args, 'sponsor')}
			}
		}
```

Reply:
i would suggest to standardize organizers and only make the sponsors customizable

Reply:
```suggestion
		Cell{name = 'Organizer(s)', content = self:_createOrganizers(args)},
		Customizable{
			id = 'sponsors',
			children = {
				Cell{name = 'Sponsor(s)', content = self:getAllArgsForBase(args, 'sponsor')}
			}
		}
```

Reply:
Done